src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,47 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
 
+        int supplyTotal = 0;
+        int buyTotal = 0;
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] parts = line.split(",");
+                String operationType = parts[0];
+                int amount = Integer.parseInt(parts[1]);
+
+                if (operationType.equals("supply")) {
+                    supplyTotal += amount;
+                } else if (operationType.equals("buy")) {
+                    buyTotal += amount;
+                }
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("An error occurred while reading the file");
```

Comment:
The catch block should include the original exception when re-throwing as a RuntimeException to preserve the stack trace and help with debugging. Follow the checklist item 'Don't ignore exceptions'.